### ISSUE TITLE
Problems with relative directory when using zip actions

### ISSUE THREAD

**Issue by csantanapr** [apache/incubator-openwhisk-runtime-docker#61]
TLDR; have the python proxy set current directory to `/action/`

The current directory is set to default meaning currently is `/actionProxy/`
When using a zip action the contents of the action is extracted to `/action/` 
If the zip file has more files and the exec binary needs to reads the file the location would be `/action/file.txt` for example. 
I think is better to have the current directory set to `/action/` this way the `exec` can assume the relative directory to `exec` or zip is the current directory hiding any implementation details of `/action` and also making testing the action locally easier before build the zip and deploying.

This is already handle in python zip action correctly via [`os.chdir`](https://github.com/apache/incubator-openwhisk-runtime-python/blob/master/core/pythonAction/pythonrunner.py#L51)
